instrumentserver/testing/dummy_instruments/generic.py

```diff
@@ -38,4 +39,49 @@ def __init__(self, name: str, *args, **kwargs):
             self.add_submodule(chan_name, channel)
 
     def test_func(self, a, b, *args, c: List[int] = [10, 11], **kwargs):
-        return a, b, args[0], c, kwargs['d'], self.param0()
\ No newline at end of file
+        return a, b, args[0], c, kwargs['d'], self.param0()
+
+class DummyInstrumentRandomNumber(Instrument):
```

Comment:
I don't understand this instrument.
how are the random numbers set? do you have to call the generate_data function somewhere?
I would've expected simply that calling the parameter would generate random numbers?

Reply:
I could not make this work by overriding the get function of this instrument. When I call it something else (not get) and hard-code the new name into the logger, the instrument works as intended. However if I leave the logger to use the generic qcodes instrument get function it goes straight into the Instrument get and not the overridden method. I could not figure out how to make it properly work. 

At the moment the logger and dashboard are technically functioning with this instrument, but it is not creating random numbers.